### DISCUSSION TITLE
we need version for linux

### DISCUSSION THREAD

**Discussion by ahmeteid7** [aws/codewhisperer-command-line-discussions#94] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Feature Details

we were waiting for fig ai to release version for linux, you will continue that?

**Comment by grant0417** [comment]
See #14, this is currently in the roadmap if you read here: https://fig.io/blog/post/fig-is-sunsetting
